tests/test_fusor.py

```diff
@@ -709,12 +709,12 @@ async def test_transcript_segment_element(
     assert tsg[0].model_dump() == expected.model_dump()
 
     expected.exon_end_offset = -5
-    expected.element_genomic_end.location.interval.start.value = 154170404
-    expected.element_genomic_end.location.interval.end.value = 154170405
+    expected.element_genomic_end.location.interval.start.value = 154170405
```

Comment:
As per discussion in line 263, I'm unsure if the values should be incremented by 1